### ISSUE TITLE
[TODO] Produce wheels with `musllinux` tags

### ISSUE THREAD

**Issue by webknjaz** [ansible/pylibssh#233]
##### SUMMARY

Recently, a new PEP 656 has been accepted. It allows producing platform-specific wheels that can be unpacked under Alpine.

We should integrate producing and testing them into our CI/CD once there's enough ecosystem support in place.

##### ISSUE TYPE
- Feature Idea

##### ADDITIONAL INFORMATION

Blocked on https://github.com/pypa/auditwheel/issues/305.

**Comment by mcassaniti** [comment]
It seems https://github.com/pypa/auditwheel/issues/305 is now closed. Is this issue still blocked?

**Comment by webknjaz** [comment]
No, that issue doesn't block this. But some infra work might be needed.
